### DISCUSSION TITLE
fwd http to https

### DISCUSSION THREAD

**Discussion by dennisscg** [netbox-community/netbox#11637] [Q&A] [answered]
Can someone please point me in the right direction on how to setup a forward from http to https for netbox ? Thank you in advance.

I cant seem to find any documents that explain how to do that.

DS

**Comment by jeremystretch** [comment, marked as answer]
NetBox itself doesn't do this, but you can easily configure your frontend HTTP server to automatically redirect HTTP requests. Here's an [nginx config](https://github.com/netbox-community/netbox/blob/a137cd6cbefa4654cc8f69c367b3258d3ccdadaa/contrib/nginx.conf#L24-L29) for example.
